src/Create.js

```diff
@@ -1,10 +1,85 @@
 import React, { Component } from 'react';
+import { getTypes, createDessert } from './fetch-utils.js';
+import classNames from 'classnames';
 
 // const URL = 'https://aqueous-coast-92629.herokuapp.com/desserts';
 
 class Create extends Component {
+    state = {
+            name: '',
+            icing: '',
```

Comment:
Remember, the initial state value you save here should always match the data type of the values you'll be saving throughout your app. In this case, `icing` is a boolean, so you should initialize this as either `true`/`false` depending on which you prefer.